index.js

```diff
@@ -66,15 +66,15 @@ function paginate(query, options, callback) {
   return Promise.all(promises).then((data) => {
     let result = {
       docs: data.docs,
-      total: data.count,
+      total: data.countDocuments,
       limit: limit
     };
     if (offset !== undefined) {
       result.offset = offset;
     }
     if (page !== undefined) {
       result.page = page;
-      result.pages = Math.ceil(data.count / limit) || 1;
```

Comment:
This should stay as is (_data.count_). Because property count is result of **@estimatedDocumentCount** method.